ocs/ocp.py

```diff
@@ -154,3 +154,20 @@ def wait_for_resource_status(
             if sample.status.phase == condition:
                 return True
         return False
+
+    # this method location is temporary, will be changed to ocs class
+    def exec_ceph_cmd(self, ceph_cmd):
```

Comment:
Maybe you can make it more generic and implement 'exec_ccmd_on_pod'?
I'm not sure what are the differences between tools pod and mon pod for example in terms of executing commands but in case it is not that different maybe worth to handle it

Reply:
@RazTamir see new commit